<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE log SYSTEM "logger.dtd">
<log>
<record>
  <date>2023-01-11T22:12:06.297392700Z</date>
  <millis>1673475126297</millis>
  <nanos>392700</nanos>
  <sequence>0</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>1</thread>
  <message>El usuario no cuenta con ningún token, pues el fichero archivos/token.dat no existe</message>
</record>
<record>
  <date>2023-01-11T22:12:06.616327800Z</date>
  <millis>1673475126616</millis>
  <nanos>327800</nanos>
  <sequence>1</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>1</thread>
  <message>El dispositivo no cuenta con ningún usuario asociado</message>
</record>
</log>
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<!DOCTYPE log SYSTEM "logger.dtd">
<log>
<record>
  <date>2023-01-11T22:13:22.357442400Z</date>
  <millis>1673475202357</millis>
  <nanos>442400</nanos>
  <sequence>0</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>1</thread>
  <message>El usuario no cuenta con ningún token, pues el fichero archivos/token.dat no existe</message>
</record>
<record>
  <date>2023-01-11T22:13:22.493474500Z</date>
  <millis>1673475202493</millis>
  <nanos>474500</nanos>
  <sequence>1</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>1</thread>
  <message>El dispositivo no cuenta con ningún usuario asociado</message>
</record>
<record>
  <date>2023-01-11T22:13:27.649763400Z</date>
  <millis>1673475207649</millis>
  <nanos>763400</nanos>
  <sequence>2</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>21</thread>
  <message>Libreria de la base de datos cargada con exito</message>
</record>
<record>
  <date>2023-01-11T22:13:46.489389Z</date>
  <millis>1673475226489</millis>
  <nanos>389000</nanos>
  <sequence>3</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>21</thread>
  <message>El usuario Anderelmakina2 existe</message>
</record>
<record>
  <date>2023-01-11T22:13:46.497396800Z</date>
  <millis>1673475226497</millis>
  <nanos>396800</nanos>
  <sequence>4</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>21</thread>
  <message>La contraseña introducida para el usuario Anderelmakina2 es correcta</message>
</record>
<record>
  <date>2023-01-11T22:13:46.530401100Z</date>
  <millis>1673475226530</millis>
  <nanos>401100</nanos>
  <sequence>5</sequence>
  <logger>BD users</logger>
  <level>INFO</level>
  <class>gestionUsuarios.GestionUsuarios</class>
  <method>log</method>
  <thread>21</thread>
  <message>Usuario asociado al ID -510543102 encontrado y retornado correctamente</message>
</record>
